### PR TITLE
feat: better descriptive naming for debt contracts

### PR DESCRIPTION
**Pull request by manuelpolzhofer**

### CODE REVIEW

File: src/core/debt_register.sol [hunk 1]
```diff
@@ -105,71 +107,70 @@ contract Beans is DSNote {
...
+            rates[rate].lastUpdated = uint48(now);
+            incTotalDebt(rate, wad);   
         }
     }
 
-    function burden(uint loan, uint fee) public view returns (uint) {
-        uint chi = fees[fee].chi;
-        if (now >= fees[fee].rho) {
-            (chi, ,) = compounding(fee);
+    function getCurrentDebt(uint loan, uint rate) public view returns (uint) {
```
Comment (charleenfei) [inline src/core/debt_register.sol]: we don't actually use this function anywhere

Comment (manuelpolzhofer) [reply src/core/debt_register.sol]: We use it in the UI :)

Comment (charleenfei) [reply src/core/debt_register.sol]: 🙈

File: src/core/debt_register.sol [hunk 2]
```diff
@@ -29,30 +29,32 @@ contract Beans is DSNote {
...
     // --- Data ---
     // https://github.com/makerdao/dsr/blob/master/src/dsr.sol
-    struct Fee {
-        uint debt;
-        uint chi;
-        uint speed; // Accumulation per second
-        uint48 rho; // Last time the rate was accumulated
+    struct Rate {
+        uint debt;  // Total debt of all loans with this rate
+        uint rateIndex; // Accumulated rates
```
[the remark this answers is not in the document]
Comment (manuelpolzhofer) [reply src/core/debt_register.sol]: I will fix it

**Pull request merged**